yarn.lock

```diff
@@ -5003,7 +5343,7 @@ level-js@^2.2.4:
     typedarray-to-buffer "~1.0.0"
     xtend "~2.1.2"
 
-level-js@timkuijsten/level.js#idbunwrapper:
+"level-js@github:timkuijsten/level.js#idbunwrapper":
```

Comment:
Weird, somehow this changed from being a npm link (I think?) to being a github link, requiring git and therefore breaking the docker image.

Reply:
Fixed it in 1a968ee. More context: 

> Same thing happened to me after a `yarn cache clean` (trying to get a git branch dependency to update).
> 
> Removing the `node_modules` folder and running `yarn` again seemed to work around the issue.
> – https://github.com/yarnpkg/yarn/issues/2083#issuecomment-272424677